src/pages/alternatives.js

```diff
@@ -2,16 +2,53 @@ import React from 'react';
 import Link from 'gatsby-link';
 import Img from 'gatsby-image';
 
+import PrimaryButton from '../components/PrimaryButton';
+
+import styles from './alternatives.module.scss';
+import sharedStyles from '../assets/shared-styles/shared.module.scss';
+import classNames from 'classnames';
```

Comment:
This is just a convention, and doesn't affect the code much but is good to know: when doing `import`s at the top of a file, generally you group external libraries at the top. So this `classNames` library you added would go above the link & img import statements. Then the convention is to have a new line and then import the things from your own code (like the Primary button like you did here!). So it's not required but you could move the classnames import statement up to the top of the file just for organization!

Reply:
Oh nice to know, thanks for pointing that out! :-)